cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -94,6 +99,10 @@
         display: block;
         width: 100%;
     }
+    &_title {
+        color: @black;
+        .u-link-child__hover(@pacific-80);
```

Comment:
Could this be simplified by moving the link into the heading and using the color utility?

```
&_title a {
  .u-link__colors( @black, @pacific-80 );
}